src/ExNavigationTypeDefinition.js.flow

```diff
@@ -40,6 +40,7 @@ export type ExNavigationBarConfig = {
   titleStyle?: Object | number,
   tintColor?: string,
   backgroundColor?: string | Function,
+  backgroundImage?: string | Function,
```

Comment:
I think `backgroundImage` prop is not very useful for all cases. what about just a `background` prop where you can pass any react component including image?